### ISSUE TITLE
nodate in DefineBibliographyStrings or DeclareLabeldate

### ISSUE THREAD

**Issue by Oberreiter** [plk/biblatex#515]
Hi,
I've tried to use `\DeclareLabeldate` to print "o. J." when no year is given.
```
\DeclareLabeldate{
  \field{date}
  \field{year}
  \field{eventdate}
  \field{origdate}
  \field{urldate}
  \literal{o\adddot\,J\adddot}
}
```
But the code fails because there are no special character allowed in literal.

So I tried to change `\DefineBiblographyStrings{naustrain}`
```
\DefineBibliographyStrings{naustrian}{
 nodate={o\adddot\,J\adddot}
}
```
But this time I get "o. J." in the Bibliography as expected. But with the authoryear cite Metode I get "Autor, nodate"

See the mini Example:

```
\documentclass[12pt,oneside,naustrian,a4paper]{scrbook}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[a4paper]{geometry}
\usepackage{babel}
\usepackage{setspace}

%Bibber Biblatex
\usepackage{filecontents}
\begin{filecontents}{biblatex.cfg}
  \NewBibliographyString{noauthor}

  \DefineBibliographyStrings{naustrian}{
    noauthor      =   {N\adddot\,N\adddot}
  }
  \newcommand*\nosomethings{noauthor}
  \@for \xx:=\nosomethings \do {
    \expandafter\ifcsname\xx\endcsname\relax\else
    \expandafter\expandafter\expandafter\expandafter\edef\csname\xx\endcsname{\noexpand\bibstring{\xx}}%
    \fi}
\end{filecontents}
\usepackage[bibencoding=utf8,style=authoryear,natbib=true,backend=biber,dashed=false]{biblatex}

\usepackage{filecontents}
\begin{filecontents}{bib.bib}
@Book{nono,
  title = {no Author no Year},
  author = {\noauthor}
}
\end{filecontents}

\addbibresource{bib.bib}
%\DeclareLabeldate{
%  \field{date}
%  \field{year}
%  \field{eventdate}
%  \field{origdate}
%  \field{urldate}
%  \literal{o\adddot\,J\adddot}
%}
\DefineBibliographyStrings{naustrian}{
 nodate={o\adddot\,J\adddot}
}

\begin{document}

\cite{nono}
\printbibliography

\end{document}
```
 
Kind regards
Andreas

**Comment by plk** [comment]
This is more of a use problem - can you ask this on tex.stackexchange.com

**Comment by moewew** [comment]
I think this should be fixed in 3.7/2.7, see https://github.com/plk/biblatex/issues/494 (there should also be another ticket for this, maybe I can find it).

**Comment by Oberreiter** [comment]
Thx
